RELEASE_NOTES.md

```diff
@@ -1,7 +1,16 @@
 # Release Notes
 
+## Version TBD
+_Date TBD_
+
+### Data Modifications
+* Update g1 and g2 macra measures to their new mappings
+
+---
+
 ## Version 14.0.0
 _23 April 2018_
+>>>>>>> 514aa1a8ced7542c2221ef7593bf0736b81bf015
```

Comment:
None of this should be in here. All "new" RELEASE NOTES stuff should be in the PENDING_RELEASE_NOTES file, or an OCD-specific file. Also, line 13 is a merge conflict line; *definitely* shouldn't be here